src/bblock/system.cpp

```diff
@@ -148,7 +150,7 @@ void System::AddMonomerInfo() {
 
 }
 
-void System::AddDimers() {
+void System::AddDimersAndTrimers() {
```

Comment:
You might want to have this as a `add_clusters` where it takes a `n` argument depend on how you want to go.

Reply:
Will do. Thanks!